File: 백준/Bronze/24544. 카카오뷰 큐레이팅 효용성 분석/카카오뷰 큐레이팅 효용성 분석.c
#include <stdio.h>

int main() {
    int N;
    scanf("%d", &N);
    int As[N], Bs[N];
    for (int i = 0; i < N; i++) {
        scanf("%d", &As[i]);
    }
    for (int i = 0; i < N; i++) {
        scanf("%d", &Bs[i]);
    }
    int totalSum = 0;
    for (int i = 0; i < N; i++) {
        totalSum += As[i];
    }
    printf("%d\n", totalSum);
    int tmp = 0;
    for (int i = 0; i < N; i++) {
        tmp += ((Bs[i] ^ 1) * As[i]);
    }
    printf("%d\n", tmp);
    return 0;
}

File: 백준/Bronze/24544. 카카오뷰 큐레이팅 효용성 분석/카카오뷰 큐레이팅 효용성 분석.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int N = sc.nextInt();
        int[] As = new int[N];
        int[] Bs = new int[N];
        for (int i = 0; i < N; i++) {
            As[i] = sc.nextInt();
        }
        for (int i = 0; i < N; i++) {
            Bs[i] = sc.nextInt();
        }
        int totalSum = 0;
        for (int a : As) {
            totalSum += a;
        }
        System.out.println(totalSum);
        int tmp = 0;
        for (int i = 0; i < N; i++) {
            tmp += ((Bs[i] ^ 1) * As[i]);
        }
        System.out.println(tmp);
    }
}